packages/atomic/src/components/insight/atomic-insight-interface/atomic-insight-interface.new.stories.tsx

```diff
@@ -1,31 +1,76 @@
-/* eslint-disable @cspell/spellchecker */
-
+import {getSampleInsightEngineConfiguration} from '@coveo/headless/insight';
 import type {Meta, StoryObj as Story} from '@storybook/web-components';
-import {html} from 'lit';
 import {parameters} from '@/storybook-utils/common/common-meta-parameters';
 import {renderComponent} from '@/storybook-utils/common/render-component';
-import {wrapInInsightInterface} from '@/storybook-utils/insight/insight-interface-wrapper';
+import '@/src/components/common/atomic-icon/atomic-icon'; // Import atomic-icon for pager buttons
+import './atomic-insight-interface.ts'; // Direct import for Lit component
+import type {AtomicInsightInterface} from './atomic-insight-interface';
+
+async function initializeInsightInterface(canvasElement: HTMLElement) {
+  await customElements.whenDefined('atomic-insight-interface');
+  const insightInterface = canvasElement.querySelector(
+    'atomic-insight-interface'
+  ) as AtomicInsightInterface;
 
-const {decorator, play} = wrapInInsightInterface();
+  await insightInterface.initialize(getSampleInsightEngineConfiguration());
+}
 
 const meta: Meta = {
   component: 'atomic-insight-interface',
-  title: 'Atomic/Insight/InsightInterface',
+  title: 'Insight/Interface',
   id: 'atomic-insight-interface',
-
   render: renderComponent,
-  decorators: [decorator],
   parameters,
-  play,
+  play: async (context) => {
+    await initializeInsightInterface(context.canvasElement);
+    const insightInterface = context.canvasElement.querySelector(
+      'atomic-insight-interface'
+    );
+    await insightInterface!.executeFirstSearch();
+  },
+  argTypes: {
+    'attributes-language': {
+      name: 'language',
+      type: 'string',
+      table: {
+        category: 'attributes',
+      },
+    },
+    'slots-default': {
+      name: 'default',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'slots',
+      },
+    },
+    'slots-full-search': {
+      name: 'full-search',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'slots',
+      },
+    },
```

Comment:
This is necessary because the component is internal, so cem doesn't seem to be detecting the slots even though they have JS Doc